README.md

```diff
@@ -679,6 +684,38 @@ if [[ $? -ne 0 ]]; then
 fi
 ```
 
+### Use a `pre-commit` hook
+
+The [pre-commit](https://pre-commit.com/) library allows hooks to be declared within a `.pre-commit-config.yaml` configuration file in the repo, and therefore more easily maintained across a team.
+
+Users of pre-commit can simply add `standard` to their `.pre-commit-config.yaml` files:
+```yaml
+  - repo: https://github.com/standard/standard
+    rev: master
+    hooks:
+      - id: standard
+```
+
+Alternatively, for more advanced styling configurations (like plugins for React packages), use `standard` as a plugin within an `eslint` configuration:
+```yaml
+  - repo: https://github.com/pre-commit/mirrors-eslint
+    rev: master
+    hooks:
+      - id: eslint
+        files: \.[jt]sx?$  # *.js, *.jsx, *.ts and *.tsx
+        types: [file]
+        additional_dependencies:
+          - eslint@latest
+          - eslint-plugin-standard@latest
```

Comment:
We don't use `eslint-plugin-standard` anymore. And I don't see `eslint-config-standard` included here.

I'd rather keep this section simple, tbh, and just remove the instructions for using a custom eslint config.

Reply:
@feross that explains a few things about my configuration, and to my mind serves as evidence that it actually should be in there. The unbelievable agony of figuring all this stuff myself would've been quickly bypassed by this useful snippet!

However, hearing your point I've made a second attempt, corrected plugin-->config and removed all the non-standard-related plugins (a helpful code snippet but perhaps not for here.

What do you think? If you still don't like it I'll remove it. But I honestly feel like that would waste much of the effort I went to!

Reply:
Looks good now